features/email_confirmation.feature

```diff
@@ -0,0 +1,17 @@
+Feature: Email confirmation
+  As a guidance specialist
+  I want to confirm that I have the correct email address for the customer
+  to ensure an email is sent successfully
```

Comment:
I think this is better written the way you explained it in showcase and potentially written on behalf of Pension Wise, as we're the people who care about delivery failures...

```
As Pension Wise
I want to ensure that the customer's email address has been entered correctly
So the appointment summary reaches the customer by email
```

Something like that?